### ISSUE TITLE
PointGroup forward pass issue

### ISSUE THREAD

**Issue by accoumar12** [Pointcept/Pointcept#335]
Hello, I successfully applied semantic segmentation on my dataset, and i would like to try out instance segmentation which seems better fitted to my usecase.
I do not understand why self.bias_head should take as input feat and not feat["feat"]. When using feat, i get an error since it is supposed to take as argument a tensor to my comprehension.
This is how i corrected the code:  
  
    def forward(self, data_dict):
        coord = data_dict["coord"]
        segment = data_dict["segment"]
        instance = data_dict["instance"]
        instance_centroid = data_dict["instance_centroid"]
        offset = data_dict["offset"]

        feat = self.backbone(data_dict)
        bias_pred = self.bias_head(feat["feat"])
        logit_pred = self.seg_head(feat["feat"])

Thanks !

**Comment by Gofinge** [comment]
Oh, are you using PTv3? We introduce a structure call "Point" in PTv3, and since PTv3, backbone return Point instead of feature. I will optimize this later.

**Comment by accoumar12** [comment]
Yeah I am using PTv3, so I need to retrieve the features to feed the bias and segmentation heads. Also, I was wondering why you do not implement the full pipeline of PointGroup, notably the dual set approach. Is there any motivation to do so ? Have you also tried others more recent instance frameworks ? Thanks a lot and keep up the good work !

**Comment by Gofinge** [comment]
Haha, save some time. I am majorly working on pre-training and backbone. I only need a simple baseline as PointGroup to help me verify the effectiveness of these things. Hope our community can help me with these things.

**Comment by fabioscap** [comment]
Hello,
Are there any changes regarding this? as of today, PointTransformerV3 and PointGroup are still not compatible. Are you planning on fixing this or should I try to open a PR?

**Comment by Gofinge** [comment]
Sorry, for super late reponse. I am working on it now.

**Comment by Gofinge** [comment]
Alright, solved here: https://github.com/Pointcept/Pointcept/commit/58e101fe17c11527e587f5476bdd0685667e3dfa
